src/queries.py

```diff
@@ -82,9 +80,9 @@ def with_params(
     ),
     "PERIOD_TOTALS": QueryData(
         name="Accounting Period Totals",
-        filepath=dashboard_file("period-totals.sql"),
-        v1_id=448457,
-        v2_id=-1,  # Not implemented
+        filepath="period_totals.sql",
```

Comment:
These files (queries) introduced here - did not exist previously within the main project code (it was only ever part of the dashboard directory) because it is not actually used for the computation of solver payouts. However, this query does live on the main dashboard. Including them here - now is merely to compensate for the fact that we are essentially losing the dashboard management functionality (however the dashboard directory has not yet been removed from the project).